src/applications/vaos/tests/list/pending-appointments.unit.spec.jsx

```diff
@@ -289,6 +289,66 @@ describe('VAOS integration: pending appointments', () => {
       expect(baseElement).to.contain.text('patient.test@va.gov');
       expect(baseElement).to.contain.text('5555555566');
     });
+
+    it('should show video request', async () => {
```

Comment:
We were missing a test for video requests, which is useful because there's some logic in the transformer to properly set the coding for those.